R/convert_to_incidence.R

```diff
@@ -0,0 +1,17 @@
+# Function to convert to incidence
+convert_to_incidence <- function(sim) {
+  incidence_data <- incidence2::incidence(sim,
+                                          date_index = c("date_onset", "date_death", "date_recovery"),
+                                          interval = 1L) %>%
+    complete_dates() %>%
```

Comment:
I think the newest version of {incidence2} allows you to set `complete_dates` within the `incidence2::incidence()` call.